style.css

```diff
@@ -0,0 +1,635 @@
+  }
+
+  .main-page {
+    margin: 0;
+  }
+
+  .top-bg {
+    background: #272a31;
+  }
+
+  .fa-brands {
+    color: #fff;
+  }
+
+  .top {
+    display: flex;
+    color: #fff;
+  }
+
+  .lg-logo {
+    display: flex;
+    padding: 0 10px;
+  }
+
+  .logo {
+    padding: 5px;
+    font-weight: 700;
+    display: block;
+    text-decoration: none;
+    color: #272a31;
+    text-align: center;
+  }
+
+  .list {
+    display: flex;
+    gap: 20px;
+    list-style: none;
+    font-weight: 300;
+    color: #272a31;
+    padding: 5px;
+    align-items: center;
+  }
+
+  .list-li {
+    color: #ec5242;
+    list-style: none;
+    text-decoration: none;
+  }
+
+  .btn {
+    border: 5px solid #ec5242;
+    color: #ec5242;
+    display: flex;
+    font-size: 15px;
+    font-weight: 500;
+    padding: 5px;
+    text-decoration: none;
+  }
+
+  .nav-1 {
+    gap: 5%;
+    padding: 0;
+    margin: 0;
+  }
+
+  .first {
+    font-weight: 400;
+    padding-top: 10px;
+    color: #ec5242;
+    margin: 0;
+  }
+
+  .second {
+    font-weight: 900;
+    color: #ec5242;
+    margin: 20px;
+    font-size: 25px;
+    margin-bottom: 15px;
+  }
+
+  .second-pr {
+    margin: 20px;
+    font-weight: 400;
+    color: #272a31;
+    margin-bottom: 15px;
+  }
+
+  .dates {
+    font-weight: 700;
+    margin: 20px;
+  }
+
+  .menu {
+    cursor: pointer;
+    align-self: center;
+    display: none;
+  }
+
+  .bar {
+    width: 15px;
+    margin: 4px auto;
+    background-color: black;
+    height: 2px;
+    display: none;
+  }
+}
+
+.card-section {
+  background-color: #272a31;
+  margin: 0;
+  padding: 20px 0;
+}
+
+.card-main {
+  align-content: center;
+  color: #fff;
+  text-align: center;
+  margin: 0;
+}
+
+.cards {
+  display: flex;
+  flex-direction: column;
+  gap: 20px;
+  padding: 6px;
+  justify-content: center;
+  margin-bottom: 10px;
+}
+
+.cd-card {
+  display: flex;
+  gap: 20px;
+  background: #6c757d;
+  width: 100%;
+  align-items: center;
+}
+
+.fa-solid {
+  font-size: 30px;
+  width: 10%;
+}
+
+.fa-brands {
+  font-size: 30px;
+  width: 10%;
+}
+
+.cards-hd {
+  color: #ec5242;
+  font-size: 15px;
+  width: 30%;
+}
+
+.cards-pr {
+  color: #fff;
+  font-size: 2px;
+  font-weight: 100;
+  width: 55%;
+}
+
+.main-btn-2022 {
+  text-align: center;
+}
+
+.card-btn {
+  color: #fff;
+  text-align: center;
+  display: none;
+}
+
+.btn-2022 {
+  background: #ec5242;
+  padding: 10px;
+}
+
+@media (min-width: 768px) {
+  .card-section {
+    margin: 0;
+  }
+
+  .cards {
+    display: flex;
+    flex-direction: row;
+  }
+
+  .cd-card {
+    display: flex;
+    flex-direction: column;
+    gap: 20px;
+    width: 18%;
+    padding-top: 10px;
+  }
+
+  .btn-2022 {
+    display: none;
+  }
+}
+
+.speakers-section {
+  background-color: #d3d3d3;
+}
+
+.spks-main {
+  display: flex;
+  flex-direction: column;
+}
+
+.speakers {
+  display: flex;
+  margin: 0;
+  align-items: center;
+  padding: 0;
+  flex-direction: row;
+  gap: 14px;
+  width: 100%;
+}
```

Comment:
- [ ] Dear @monicah79, I see you made changes according to the previous reviewer, Yet the display is not appropriate yet. please consider using the right font size and type to make the display easy to read.

## Figma design 
![speakers-2](https://user-images.githubusercontent.com/69077061/204107622-edd59e49-b13b-4f05-9fda-ff8d73e5639a.PNG)

## Your design
![speakers](https://user-images.githubusercontent.com/69077061/204107621-bc420dae-d4a0-45aa-bda6-bc72e06b1d1d.PNG)

Reply:
- [ ] As the previous reviewer mentioned that kindly you should use the right font size and type to make the display easy to read.